CHANGELOG.md

```diff
@@ -8,6 +8,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Tractus-X has switched to YY.0M.MICRO [Calendar Versioning](https://calver.org/) for overarching, quarterly Releases. Semantic Versioning remains unchanged for all components.
 
+## [24.03] - 2024-03-XX
+
+### Added
```

Comment:
PURIS
Industry Core KIT
Environmental and Social Standards (ESS) KIT
Manufacturing as a Service (MaaS) KIT
Certificate Management KIT